build.gradle

```
@@ -20,6 +20,9 @@ dependencies {
 	// log
 	implementation 'net.rakugakibox.spring.boot:logback-access-spring-boot-starter:2.7.1'
 
+	// validate
+	implementation group: 'org.springframework.boot', name: 'spring-boot-starter-validation', version: '2.7.0'
```

```suggestion
	implementation group: 'org.springframework.boot', name: 'spring-boot-starter-validation'
```

어떤 차이가 있을까요? 나중에 스프링이 의존성을 어떻게 관리하는지 학습해보면 좋을 것 같아요! ☺️ 지금은 그냥 넘어가도 좋습니다.

---

우선 수정해 놓고 의존성 관리에 대해 추후 학습해 보겠습니다!🙏🏻